siriuspy/siriuspy/devices/ids.py

```diff
@@ -419,7 +572,7 @@ def _write_sp(self, propties_sp, values, timeout=None):
                 propty_rb, value, timeout=timeout, comp='eq')
         return success
 
-    def _wait(self, props_values, timeout=None, comp='eq'):
+    def _local_wait(self, props_values, timeout=None, comp='eq'):
```

Comment:
Also, change the name of the method to something more intuitive in relation to its operation. Maybe `_wait_propts` or something similar.

remember to update timeout so that the total time is not larger than the user defined timeout.

Reply:
I think we then should also change to base class `Device._wait`. I just replicated the name we already use, by locally.

Reply:
changed.

Reply:
implemented `timeout` as total timeout.